airbyte-integrations/connectors/source-rd-station-marketing/source_rd_station_marketing/streams.py

```diff
@@ -0,0 +1,203 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+from abc import ABC
+from datetime import date
+from typing import Any, Iterable, Mapping, MutableMapping, Optional
+
+import pendulum
+import requests
+from airbyte_cdk.sources.streams.http import HttpStream
+
+
+class RDStationMarketingStream(HttpStream, ABC):
+    url_base = "https://api.rd.services"
+    primary_key = None
+    page = 2
+    page_size=125
+    extra_params = {}
+    data_field = None
+
+    def __init__(self, authenticator, replication_start_date=None, **kwargs):
+        super().__init__(authenticator=authenticator, **kwargs)
+        self._replication_start_date = replication_start_date
+
+    def path(self, **kwargs) -> str:
+        class_name = self.__class__.__name__
+        return f"/platform/{class_name[0].lower()}{class_name[1:]}"
```

Comment:
Do you mind add an unit test for this one?

Reply:
Sure, added one!